package.json

```diff
@@ -107,18 +107,18 @@
     "karma-source-map-support": "^1.2.0",
     "karma-sourcemap-loader": "^0.3.6",
     "karma-webpack": "^2.0.3",
-    "mocha": "^3.0.1",
+    "mocha": "^5.0.1",
     "npm-run-all": "^4.0.0",
     "phantomjs-prebuilt": "^2.1.7",
     "rimraf": "^2.5.3",
-    "rollup": "^0.48.2",
+    "rollup": "^0.56.3",
     "rollup-plugin-babel": "^3.0.2",
-    "rollup-plugin-memory": "^2.0.0",
+    "rollup-plugin-memory": "^3.0.0",
     "rollup-plugin-node-resolve": "^3.0.0",
-    "sinon": "^2.2.0",
+    "sinon": "^4.4.2",
```

Comment:
That's awesome! Thanks a lot for your PR 🎉 

The CI reports quite a few deprecation messages like:

```
INFO: 'sinon.reset is deprecated and will be removed from the public API in a future version of sinon.'
```

Perhaps we can fix these too?

Reply:
will do